CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
       - Fixed an issue that would detect turning circles as sliproads
       - Fixed a bug where post-processing instructions (e.g. left + left -> uturn) could result in false pronunciations
       - Fixes a bug where a bearing range of zero would cause exhaustive graph traversals
+    - Infrastructure:
+      - Adds a feature to limit results in nearest service with a default of 100 in `osrm-routed`
```

Comment:
What do you mean with abi change?

`libosrm`'s `EngineConfig` now provides a new limit ([unlimited by default](https://github.com/Project-OSRM/osrm-backend/blob/a618f028fb802ceea67c1a4df189b92b4758e91a/include/engine/engine_config.hpp#L69)). It's only the `osrm-routed` binary setting the limit of `100` (and the user can still change this with [`--max-nearest-size`](https://github.com/Project-OSRM/osrm-backend/blob/a618f028fb802ceea67c1a4df189b92b4758e91a/src/tools/routed.cpp#L105-L107).

We don't guarantee ABI stability at all:
https://github.com/Project-OSRM/osrm-backend/blob/master/docs/releasing.md

Only API stability. So adding a new member in the `libosrm` header should be fine.